roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Install nfs-utils
+  yum:
+    pkg: nfs-utils
+    state: present
+
+- name: Ensure exports directory exists
+  file:
+    path: "{{ exports_dir }}"
+    state: directory
+
+- name: Ensure export directories exist
+  file:
+    path: "{{ exports_dir }}/{{ item }}"
+    state: directory
+    mode: 0777
+    owner: nfsnobody
+    group: nfsnobody
+  with_items:
+  - "{{ registry_volume }}"
+
+- name: Configure exports
+  template:
+    dest: /etc/exports
+    src: exports.j2
+  notify:
+  - restart nfs-server
+
+- name: Enable and start services
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+  register: start_result
+  with_items:
+  - nfs-server
+
+- name: Pause 30 seconds if services were just restarted
+  pause: seconds=30
+  when: "{{ True in (start_result.results
+                      | map(attribute='changed')
+                      | list) }}"
+
+- name: Configure nfs settings
+  lineinfile:
+    dest: /etc/sysconfig/nfs
+    regexp: "{{ item.regex }}"
+    line: "{{ item.line }}"
+    create: yes
+  with_items:
+  - regex: '^RPCMOUNTOPTS='
+    line: 'RPCMOUNTDOPTS="-p 20048"'
+  - regex: '^STATDARG='
+    line: 'STATDARG="-p 50825"'
```

Comment:
Also, anything you configure after the service step will cause an unnecessary restart of the nfs service on first install, I would move all of the config options above the service task.